eks/cdk/cw.py

```diff
@@ -28,8 +28,8 @@ def __init__(self, app: core.App, id: str, props, **kwargs) -> None:
                 namespace = "ContainerInsights",
                 metric_name = "node_filesystem_utilization",
                 statistic = "p90",
-                period = core.Duration.seconds(30),
-                dimensions = dict(
```

Comment:
**dimensions** parameter is deprecated.
https://docs.aws.amazon.com/cdk/api/v2/python/aws_cdk.aws_cloudwatch/Metric.html#metric